packages/client/hmi-client/src/model-representation/petrinet/nested-petrinet-renderer.ts

```diff
@@ -129,6 +141,25 @@ export class NestedPetrinetRenderer extends PetrinetRenderer {
 					.attr('x2', -d.width * 0.5 + position)
 					.attr('stroke', '#ffffffcf');
 			}
+
+			transitionMatrix.forEach((row) => {
+				row.forEach((col) => {
+					if (col.value) {
+						transitionNode
+							.append('rect')
+							.attr('width', d.width / transitionMatrixLen)
+							.attr('height', d.width / transitionMatrixLen)
+							.attr('x', -d.width * 0.5 + (d.width / transitionMatrixLen) * col.row)
+							.attr('y', -d.width * 0.5 + (d.width / transitionMatrixLen) * col.col)
+							.attr('rx', '2')
+							.attr('ry', '2')
+							.style('fill', d.data.strataType ? getNodeTypeColor(d.data.strataType) : '#7fffd4aa')
+							.style('cursor', 'pointer')
+							.attr('stroke', '#ffffffcf')
+							.attr('stroke-width', 1);
```

Comment:
A couple of minor things
- rx/ry should be numbers, not strings, to be a bit more consistent with how we are doing D3 stuff elsewhere
- slight preference for color/stroke to be RGB instead of RGBA, specifying the alpha separately makes it a little more straightforward to do animations and transitins.

Reply:
I've just got rid of the alpha for now since it doesn't make any real visual difference at the moment.